skxray/fitting/xrf_model.py

```diff
@@ -521,11 +532,9 @@ def setup_elastic_model(self):
                                value=self.compton_param['coherent_sct_energy'].value,
                                expr='coherent_sct_energy')
 
-        elastic.set_param_hint('coherent_sct_energy',
```

Comment:
I think you can replace this whole chunk of code with

```python
for label in list_of_labels:
    elastic.set_param_hint(label,value=self.compton_param[label], expr=label)
```

which is what it looks like you are doing in the next look anyway.